definitions/tools/hla_consensus.cwl

```diff
@@ -237,7 +239,13 @@ inputs:
             position: 1
             itemSeparator: ','
             separate: false
-    clinical_hla_alleles:
+    clinical_MHCcI_alleles:
+        type: string[]?
+        inputBinding:
+            position: 2
+            itemSeparator: ','
+            separate: false
+    clinical_MHCcII_alleles:
         type: string[]?
         inputBinding:
             position: 2
```

Comment:
May want to make sure this position is unique from the class one position, since we're relying on positional parameters in the script.